react/components/EXPERIMENTAL_Table/context/loading.tsx

```diff
@@ -14,10 +14,21 @@ interface Loading {
 
 const LoadingContext = createContext<Loading>(null)
 
-export function LoadingProvider({
+export const LoadingProvider = function LoadingProvider({
```

Comment:
Why change only in this one? 🤔

Reply:
I actually don't remember why I did this 🤔